docs/contributors/write-documentation.rst

```diff
@@ -0,0 +1,185 @@
+.. _write documentation:
+
+##########################
+How to write documentation
+##########################
+
+In this document you will learn how to write good, informative, pretty and actionable documentation.
+
+It's not hard !
+
+Overview
+--------
+
+There are two major types of documentation:
+
+1. **docstrings**: your code's docstrings will be automatically parsed by the documentation sofware (`Sphinx <https://www.sphinx-doc.org>`_, more in `about shpinx`_).
+2. **Manual** documentation such as this document.
+
+**Both** are written in `ReStructured Text <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_ (``.rst``) format.
+
+Some of the great features of using Sphinx is to be able to automatically generate documentation from your code's docstrings, and to be able to link to other parts of the documentation.
+
+For instance: :meth:`~gflownet.gflownet.GFlowNetAgent.trajectorybalance_loss` or to an external function :func:`torch.cuda.synchronize()`.
+
+.. caution::
+
+    This page is about learning the syntax of ``.rst`` and Sphinx. It is not about learning how to write good, informative docstrings.
+    Refer to `Google's docstring guide <https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings>`_ for that, or
+    checkout full-fleshed examples in `Numpy style <https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy>`_ or `Google style <https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google>`_. We also have a short intro in the :doc:`next section </contributors/write-docstrings>`.
+
+Learn by example
+^^^^^^^^^^^^^^^^
+
+The next section will introduce many of the cool features of ``.rst`` + Sphinx + plugins.
+
+Click on "*Code for the example*" to look at the ``.rst`` code that generated what you are reading.
+
+.. tab-set::
+
+    .. tab-item:: Full-fleshed ``.rst`` example
```

Comment:
Do you really mean "full-fleshed" or [full-fledged](https://www.britannica.com/dictionary/full-fledged)?